src/stcal/ramp_fitting/ols_fit.py

```diff
@@ -144,6 +149,8 @@ def ols_ramp_fit_multiprocessing(
     number_slices: int
         The number of slices to partition the data into for multiprocessing.
 
+    avg_dark_current: float
```

Comment:
Remove these lines. It's no longer passed as a parameter.

Reply:
removed